doc/engineering-reference/src/climate-sky-and-solar-shading-calculations/climate-calculations.tex

```diff
@@ -44,86 +44,106 @@ \subsection{EnergyPlus Design Day Temperature Calculations}\label{energyplus-des
+
+$IR_H$ is defined as the rate of infrared radiation emitted from the sky falling on a horizontal upward-facing surface, in \si{\watt\per\m\squared}.
 
 \begin{equation}
-Horizontal\_IR = Sk{y_{emissivity}}\cdot Sigma\cdot Temperature_{drybulb}^4
+IR_H = \sigma T_{sky}^4
 \end{equation}
 
-where
-
-Horizontal\_IR = horizontal IR intensity \{W/m\(^{2}\)\}
-
-Sky\(_{emissivity}\) = sky emissivity
+\noindent where:
+\newline
 
-Sigma = Stefan-Boltzmann constant = 5.6697e-8 \{W/m\(^{2}\)-K\(^{4}\)\}
+$\sigma$ = Stefan-Boltzmann constant = \SI{5.6697e-8}{\watt\per\meter\squared\per\kelvin\tothe{4}}
 
-Temperature\(_{drybulb}\) = drybulb temperature \{K\}
+$T_{sky}$ = Effective mean sky temperature, or sky radiative temperature, \si{\kelvin}.
+\newline
 
-By default, the clear sky emissivity is calculated by the Clark \& Allen (1978) model as
+Over the years authors have proposed a fictitious quantity called the Sky Emissivity ($\epsilon_{sky}$) such that the following energy balance is satisfied:
 
 \begin{equation}
-Sky_{clear-sky-emissivity} = 0.787 + 0.764\cdot \ln \frac {T_{dewpoint}}{T_{Kelvin}}
+IR_H = \epsilon_{sky} \cdot \sigma \cdot T_{db}^4
 \end{equation}
 
-where, T\(_{dewpoint}\) = dewpoint temperature \{K\}.
+$\epsilon_{sky}$ = sky emissivity
 
-Alternatively, the clear sky emissivity can also be calculated using different models set by the user from options using the \textbf{WeatherProperty:SkyTemperature} object, including the calibrated version of the Berdahl and Martin, Brunt and Idso models (2017), listed as follows.
-
-Calibrated Berdahl and Martin model:
-
-\begin{equation}
-Sky_{clear-sky-emissivity} = 0.711 + 0.56\cdot(T_{dewpoint} / 100) + 0.73\cdot{T_{dewpoint} / 100}^2.
-\end{equation}
+$T_{db}$ = drybulb temperature, \si{\kelvin}
+\newline
 
-Calibrated Brunt model:
+Four correlations for $\epsilon_{sky}$ under clear-sky conditions, proposed by four sets of authors, are available in EnergyPlus. The default correlation is from Clark \& Allen (1978):
 
-\begin{equation}
-Sky_{clear-sky-emissivity} = 0.52 + 0.065\cdot(P_{water-vapor})^{0.5},
-\end{equation}
+\begin{table}[hbtp]
+\centering
+\begin{tabular}{cl}
+\textbf{Author}   & \epsilon_{sky, clear} \\ \\
+Clark \& Allen    & $= 0.787 + 0.764 \ln\left(T_{dp}/273\right)$ \\ \\
+Martin \& Berdahl & $= 0.711 + 0.56 \left(T_{dp}/100\right) + 0.73  \left(T_{dp} / 100\right)^2$ \\ \\
+Brunt             & $= 0.52 + 0.065 \left(P_{wv}\right)^{0.5}$ \\ \\
+Idso              & $= 0.70 + 5.95\times10^{-5} \left(P_{wv}\right) e^{1500/T_{db}} $
```

Comment:
Hmm, the values in these equations match the [source code](https://github.com/NREL/EnergyPlus/blob/f8de75ef10f611c2f2ec106687db5849c5f4df61/src/EnergyPlus/WeatherManager.cc#L3765-L3775) for Clark & Allen, but not for the other correlations. @xuanluo113 Which is correct, the docs or the code?

Reply:
@mjwitte The code is right. Here I listed are the original equations. They should be the calibrated equations as the source code. Thanks for catching this. Can I fix it at this branch?
![image](https://user-images.githubusercontent.com/14095981/77583521-72e94600-6e9e-11ea-9de3-6a28575620f3.png)

Reply:
@mjwitte The code is right. Here I listed are the original equations. They should be the calibrated equations as the source code. Thanks for catching this. Can I fix it at this branch?
![image](https://user-images.githubusercontent.com/14095981/77583521-72e94600-6e9e-11ea-9de3-6a28575620f3.png)

Reply:
@xuanluo113 Yes, please fix in this branch. Thanks.